packages/compiler/src/experimental/realm.ts

```diff
@@ -91,11 +98,25 @@ class StateMapRealmView<V> implements Map<Type, V> {
   }
 }
 
-/** @experimental */
+/**
+ * A symbol used to access a realm's typekit.
+ *
+ * @internal
+ * @experimental
+ */
+export const REALM_TYPEKIT = Symbol.for("TypeSpec::Realm::Typekit");
```

Comment:
```suggestion
export const REALM_TYPEKIT = Symbol.for("TypeSpec.Realm.Typekit");
```

We usually use dots as a separator here I think.

Reply:
terminally Rust brained, I am